package.json

```diff
@@ -37,6 +37,9 @@
   "bugs": {
     "url": "https://github.com/angular/angularfire/issues"
   },
+  "engines": {
+    "node": "^8.13.0 || >=10.10.0"
```

Comment:
node lts version is 12, node 8 is very behind

Reply:
I used the same engine requirement as firebase-js sdk: https://github.com/firebase/firebase-js-sdk/blob/master/package.json#L9